pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSTag.java

```diff
@@ -38,6 +40,7 @@
 public class POSTag {
```

Comment:
Since this principally a demo class, please have it write to STDOUT instead of logger.